rdr_service/genomic/genomic_job_controller.py

```diff
@@ -297,41 +304,116 @@ def ingest_member_ids_from_aw1_raw_table(self, member_ids):
 
                 for r in update_recs:
 
-                    # Set RDR attributes: aw1_file_processed_id, job_run_id, gc_site
-                    self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
+                    # AW1
+                    if n == 1:
+                        self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
 
-                    # Set attributes
-                    member = self.set_aw1_attributes_from_raw(r)
+                        self.set_aw1_attributes_from_raw(r)
 
-                    s.merge(member)
-                    completed.append(member.id)
+                    # AW2
+                    else:
+                        self.preprocess_aw2_attributes_from_raw(r, file_proc_map)
 
-        # Output results
-        result_msg = ''
-        result_msg += 'Ingestion From Raw Results:'
-        result_msg += f'    Updated IDs: {completed}'
-        result_msg += f'    Missing IDs: {missing}'
-        result_msg += f'    Multipls found for IDs: {multiples}'
+                        metrics_obj = self.set_validation_metrics_from_raw(r)
 
-        return result_msg
+                        metrics_obj = s.merge(metrics_obj)
+                        s.commit()
+                        metrics.append(metrics_obj.id)
+
+                    s.merge(r[0])
+                    completed_members.append(r[0].id)
+
+            # BQ Updates
+            if n == 2:
+                # Metrics
+                bq_genomic_gc_validation_metrics_batch_update(metrics, project_id=self.bq_project_id)
+                genomic_gc_validation_metrics_batch_update(metrics)
+
+            # Members
+            bq_genomic_set_member_batch_update(metrics, project_id=self.bq_project_id)
+            genomic_set_member_batch_update(completed_members)
+
+        return self.compile_raw_ingestion_results(completed_members, missing, multiples, metrics)
 
     def set_aw1_attributes_from_raw(self, rec: tuple):
         """
         :param rec: GenomicSetMember, GenomicAW1Raw
         :return:
         """
-        # get gc_site_id
         member, raw = rec
 
         # Iterate through mapped fields
-
         _map = raw_aw1_to_genomic_set_member_fields
 
         for key in _map.keys():
             member.__setattr__(_map[key], getattr(raw, key))
 
         return member
 
+    def set_validation_metrics_from_raw(self, rec: tuple):
```

Comment:
Is this method specific to AW2?   Make method comments more descriptive as to what raw data its using?